### PR TITLE
Eliminate coffeescript linter complaints.

### PR DESCRIPTION
**Pull request by spencern**
Almost all changes are line-length related. Any non-obvious line-length breaks were run through a coffeescript compiler to ensure identical output. 

One change is duplicate keys in the product schema.

### CODE REVIEW

File: common/collections/collections.coffee [hunk 1]
```diff
@@ -48,47 +62,58 @@ ReactionCore.Helpers.cartTransform =
...
 # Shipping
 ReactionCore.Collections.Shipping = new Mongo.Collection "Shipping"
-ReactionCore.Collections.Shipping.attachSchema ReactionCore.Schemas.Shipping
+ReactionCore.Collections.Shipping.attachSchema(
+  ReactionCore.Schemas.Shipping)
 
 # Taxes
 ReactionCore.Collections.Taxes = new Mongo.Collection "Taxes"
-ReactionCore.Collections.Taxes.attachSchema ReactionCore.Schemas.Taxes
+ReactionCore.Collections.Taxes.attachSchema(ReactionCore.Schemas.Taxes)
```
Comment (aaronjudd) [inline common/collections/collections.coffee]: linter was complaining about this, or were the parens just  added for legibility?

**Comment by aaronjudd** [comment]
Merged and closed in https://github.com/reactioncommerce/reaction-core/commit/d11a2adef1fb20c2bdc58cb3adf7799cd1d03850

File: common/collections/collections.coffee (hunk 1 above)
Comment (spencern) [inline common/collections/collections.coffee]: linter did not complain about that particular line, good catch. I added those parens to keep it consistent with all of the lines above it that we had to break to stay under 80.

Comment (aaronjudd) [inline common/collections/collections.coffee]: ok, cool I tend to include the parens when it helps legibility. keeping the parens is probably easier for those that don't want to guess the CS style.

**Pull request closed without merging**